startup/GafferSceneUI/catalogue.py

```diff
@@ -67,8 +67,16 @@ def typeIconColumnValueProvider( image, catalogue ):
 		return iconName
 
 	CatalogueUI.registerColumn(
-		"typeIcon",
-		typeIconColumnDefinition.title,
-		typeIconColumnValueProvider,
+		"status",
+		statusIconColumnDefinition.title,
+		statusIconColumnValueProvider,
 		CatalogueUI.ColumnType.Icon
 	)
+
+# Render Node column preset
+
+def renderNodeColumnValueProvider( image, catalogue ):
+	scenePlug = GafferScene.SceneAlgo.sourceScene( catalogue["out"] )
+	return scenePlug.node().getName() if scenePlug else ""
```

Comment:
I think the right approach here is to traverse up the hierarchy until there is no ancestor for which the "graphEditor:childrenViewable" metadata is False.

Alex brought up a related problem very recently too - SourceSet and SceneHistoryUI use `ancestorWithReadOnlyChildNodes()` to determine the first publicly viewable ancestor, but they overdo it by traversing out of References when it would be more useful if they showed the node within. The same "graphEditor:childrenViewable" approach would fix that.

Let's use this opportunity to kill two birds with one stone. Should we perhaps consider a more generic name than "graphEditor:childrenViewable", and add a method in MetadataAlgo to find the appropriate ancestor?

Reply:
I've dropped this column for now. We can pick that back up again in the near future if people determine `Render Node` is a useful thing. It wasn't on the original list, I just added it as we had the header around.